.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -35,3 +35,31 @@ Provide a reproducible test case that is the bare minimum necessary to generate
 ### Other info / logs
 Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks,
 please include the full traceback. Large logs and files should be attached.
+
+
+### What component(s), interfaces, languages, and integrations does this bug affect?
+Components 
+- [ ] `area/artifacts`: Artifact stores and artifact logging
+- [ ] `area/docs`: MLflow documentation pages
+- [ ] `area/examples`: Example code
+- [ ] `area/model-registry`: Model registry, model registry APIs, and the fluent client calls for
```

Comment:
Should we say something more specific than `Model registry` for the first element of this list? I'd also recommend capitalizing component names (i.e. `Model Registry`)

^ This applies to other templates as well